### PR TITLE
Edited the constitution. 

### PR DESCRIPTION
**Pull request by kgourgou**
Hi everyone, 

Here are some edits for the constitution. If you don't agree with some of them, we can talk.

Parts are to make the language a bit more clear, and others for consistency.

### CODE REVIEW

**Comment by kgourgou** [comment]
Don't merge it immediately, but take a look at [Files changed](https://github.com/UMassAmherst-GRiD/GRiD_Constitution/pull/4/files).

**Comment by kgourgou** [comment]
Guys? Anyone here?

File: constitution.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ The Executive Board will meet as a body at least once per month. Every meeting m
 
 The Co-Chair of Operations is responsible for scheduling, organizing, and running all Executive Board meetings, including taking attendance of present members and appointing an officer to take minutes.
 
+// Can members attend those meetings ??
```
Comment (markwh) [inline constitution.md:90]: I think so. At least I can't think of any reason why they shouldn't

Comment (kgourgou) [inline constitution.md:90]: Love that we can comment on specific lines. 

I can't think of any either. Perhaps we should be explicit about that?

File: constitution.md [hunk 2]
```diff
@@ -143,26 +144,32 @@ The Outreach Committee shall be headed by the Chief Outreach Officer and shall b
...
-- Maintaining a list of data science other events on campus and in the region that may be of interest to GRiD members
+- Coordinating with the Programming Committee to plan and execute networking events.
+- Maintaining a list of other data science events on campus and in the region that may be of interest to GRiD members.
 
 ## ARTICLE V: ELECTIONS AND VOTING
 <a name="elections"></a>
 ### Section A: Election 
 
-The Nomination Period for organization elections will commence at the final meeting of the fall semester and conclude at the first meeting of the spring semester. Nominations may be solicited and accepted at either meeting or in the form of email to the board during the winter break. Any member may nominate a member (including him/herself) for any Executive Board Officer position.
+The Nomination Period for organization elections will commence at the final meeting of the fall semester and conclude at the first meeting of the spring semester. Nominations may be solicited and accepted at either meeting or in the form of email to the board during the winter break. Any member may nominate any other member (including herself/him) for any Executive Board Officer position.
```
Comment (markwh) [inline constitution.md:162]: "him/herself" or "her/himself" is more accepted. Although I have become a proponent of the singular they in cases like this. I guess that would be "themself" http://blog.oxforddictionaries.com/2013/01/themself/.

**Comment by markwh** [comment]
Sorry, I guess I didn't understand the pull request protocol. Looks fine to me. See comments.

File: constitution.md (hunk 2 above)
Comment (kgourgou) [inline constitution.md:162]: Totally agree. Was going for consistency there, as it was different in other parts.

**Comment by kgourgou** [comment]
haha, no worries. 

When you have a repo, you can push at it directly and all is fine. However, what if I merge something you guys don't agree with? That is why I used the pull request which requires a review before merging. :)

**Pull request merged**